packages/waffle-chart/src/CdcWaffleChart.tsx

```diff
@@ -20,24 +20,23 @@ import { publish } from '@cdc/core/helpers/events'
 import useDataVizClasses from '@cdc/core/helpers/useDataVizClasses'
 import coveUpdateWorker from '@cdc/core/helpers/coveUpdateWorker'
 
+import { Config } from './types/Config'
+import { useWaffleChart } from './hooks/useWaffleChart'
+
 import './scss/main.scss'
 
-const themeColor = {
-  'theme-blue': '#005eaa',
-  'theme-purple': '#712177',
-  'theme-brown': '#705043',
-  'theme-teal': '#00695c',
-  'theme-pink': '#af4448',
-  'theme-orange': '#bb4d00',
-  'theme-slate': '#29434e',
-  'theme-indigo': '#26418f',
-  'theme-cyan': '#006778',
-  'theme-green': '#4b830d',
-  'theme-amber': '#fbab18'
+type CdcWaffleChartTypes = {
```

Comment:
I think CdcWaffleChartProps is a better name, up to you.

Reply:
I prefer props so I've changed that.